src/reference/antora/modules/ROOT/pages/ftp/rft.adoc

```diff
@@ -37,3 +37,23 @@ This is useful when you need to perform several high-level operations of the `Re
 For example, `AbstractRemoteFileOutboundGateway` uses it with the `mput` command implementation, where we perform a `put` operation for each file in the provided directory and recursively for its sub-directories.
 See the https://docs.spring.io/spring-integration/api/org/springframework/integration/file/remote/RemoteFileOperations.html#invoke[Javadoc] for more information.
 
+Starting with version 6.5, the `AbstractRemoteFileOutboundGateway` supports dynamic resolution of `FileExistsMode`
+at runtime via SpEL expressions. This allows you to determine the action to take when files already
+exist based on message content or other conditions.
```

Comment:
We prefer a `One sentence per line` style: https://asciidoctor.org/docs/asciidoc-recommended-practices/